keyboards/kalkulators/calc.c

```diff
@@ -0,0 +1,454 @@
+/* Copyright 2024 georgsnarbuts
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+#include <stdio.h>
+#include <stdlib.h>
+#include <math.h>
+#include <ctype.h>
+#include <string.h>
+#include "halconf.h"
+#include "tm1638.h"
+#include "usb_util.h"
+#include "wait.h"
+#include "calc.h"
```

Comment:
```suggestion
#include <stdio.h>
#include <stdlib.h>
#include <math.h>
#include <ctype.h>
#include <string.h>
#include "halconf.h"
#include "tm1638.h"
#include "usb_util.h"
#include "wait.h"
#include "kalkulators.h"
```

Reply:
Is there something to replace "#include QMK_KEYBOARD_H" with? Removing it breaks the code.

Reply:
From my comment:
`#include "kalkulators.h"`
which is what `QMK_KEYBOARD_H` is, but the macro should not be used at the keyboard level.